boards/arm/efr32mg_sltb004a/board.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020 Christian Taedcke
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include "board.h"
+#include <drivers/gpio.h>
+#include <sys/printk.h>
+
+static int efr32mg_sltb004a_init(struct device *dev)
+{
+	struct device *cur_dev;
+
+	ARG_UNUSED(dev);
+
+#ifdef CONFIG_CCS811
+	/* Enable the CCS811 power */
+	cur_dev = device_get_binding(CCS811_PWR_ENABLE_GPIO_NAME);
+	if (!cur_dev) {
+		printk("CCS811 power gpio port was not found!\n");
+		return -ENODEV;
+	}
+
+	gpio_pin_configure(cur_dev, CCS811_PWR_ENABLE_GPIO_PIN, GPIO_OUTPUT);
+	gpio_pin_set(cur_dev, CCS811_PWR_ENABLE_GPIO_PIN, 1);
```

Comment:
I think we need to get these from the board-level device tree, but I'm not quite sure how to define the bindings. A new gpio property in `dts/bindings/sensor/ams,ccs811.yaml` doesn't makes sense because this is enabling a switch to the sensor's vdd pin. @galak?

Reply:
This kind of pattern is present on most of the Silabs boards. A lot peripherals/functionalities must be enabled by pulling a pin high. This is currently done in the board initialization, but it would be really nice if this could be done via the device tree and/or something like https://elixir.bootlin.com/linux/v5.6.13/source/Documentation/devicetree/bindings/regulator/gpio-regulator.yaml would be supported.